README.md

```diff
@@ -43,6 +43,7 @@ Kubernetes is available at https://192.168.99.100:443.
 
 $ kubectl run hello-minikube --image=gcr.io/google_containers/echoserver:1.4 --hostport=8000 --port=8080
 deployment "hello-minikube" created
+#NOTE: It may take a few minutes for the echoserver pod to be up and running before you can curl it
```

Comment:
@dlorenc that's basically what I ended up doing. I found mention of it at [http://kubernetes.io/docs/getting-started-guides/minikube/](http://kubernetes.io/docs/getting-started-guides/minikube/) where it suggests `kubectl get pods --all-namespaces`